README.md

```diff
@@ -1,3 +1,149 @@
+   result:
+   ```
+   +code
+   ```
+   This is your fake ship's password, which will be used for authentication
+   from your web browser.
+8. Open a web browser and navigate to the HTTP address discussed in step 3.
+9. Paste the ship's password from step 7 into the form that appears and hit
+   enter.
+0. If the installation was successful, you should see a tile for the example
+   on the home page that appears.
+
+If the example has a React interface, further follow these instructions to
+set up a locally-hosted front-end interface:
+
+11. In a terminal session, enter the following commands (where `XYZ` is from
+    step 3):
+    ```bash
+    cd /path/to/repo-source/example-directory/ui/
+    npm install
+    echo "VITE_SHIP_URL=http://127.0.0.1:XYZ" >> .env.local
+    npm run dev
+    ```
+    The final command will generate a text digest containing a line of the form:
+    ```
+    > Local: http://localhost:ABC/apps/example-desk/
+    ```
+    The web address listed here is hosting location for your local React
+    interface. Save this string for use in the subsequent steps.
+12. Open a web browser and navigate to the web address from the step 11.
+    If the local hosting was successful, you should see the interface for the
+    example in your browser.
+13. If your browser encounters an error during the previous step, then you may
+    need to enable the [CORS] registry on your ship with the following `dojo`
+    command (where `ABC` is from step 11):
+    ```
+    |cors-approve ~~http~3a.~2f.~2f.localhost~3a.ABC
+    ```
+
+### Editing and Rebuilding ###
+
+Once the base infrastructure for an example project is in place, making changes
+is very simple.
+
+For **React front-end development**, just make sure to run `npm run dev` in a
+terminal and to visit the output "local" address for testing; the page will
+automatically update as you edit your React files.
+
+For **Sail front-end development and back-end development**, the process is only
+slightly more complicated:
+
+1. If you're editing the files in-place (i.e. in
+   `/path/to/fake-ship/example-desk/`), you can skip this step. If you're
+   editing files from the source tree
+   (i.e. in `/path/to/repo-source/example-directory/full-desk/`), you'll need
+   to copy the files to your ship with the following terminal commands:
+   ```bash
+   rm -rI /path/to/fake-ship/example-desk/*
+   cp -RL /path/to/repo-source/example-directory/full-desk/* /path/to/fake-ship/example-desk/
+   ```
+2. From within your fake ship's `dojo`, enter one of the following command
+   sequences based on your changes:
+   - If you didn't make any changes to the example app's data model:
+     ```
+     |commit %example-desk
+     ```
+   - If you did made changes to the example app's data model (e.g. `/sur` file
+     changes, `state` value changes, etc.):
+     ```
+     |nuke %example-desk
+     |commit %example-desk
+     |revive %example-desk
```

Comment:
Maybe mention `|rein` as well.

Reply:
Do you know when it would be needed for these examples? I ran through state changes requiring `|nuke`s for `%journal` and `%squad` and just `|nuke` then `|commit` seemed to work (I may just omit the `|revive` as a result; `|commit` seems to handle agent revival automatically now). Are there other workflows should I test out that would require `|nuke` then `|rein`?